# Message that provides information about the parking spot
Header header
ConvexShape vertices_map_section
ConvexShapeList vertices_illegal_areas
ConvexShape vertices_parking_slot
uint8 park_type
float64 length
float64 width
float64 area
geometry_msgs/Point midpoint